oe-init-build-env

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
+DIR=$( dirname -- "${BASH_SOURCE[0]}" )
```

Comment:
Now that I've had a second look I think we should actually copy the `zsh` compatibility code from poky, as `zsh` is kind of widespread:

```suggestion

if [ -n "${BASH_SOURCE[0]}" ]; then
    THIS_SCRIPT="${BASH_SOURCE[0]}"
elif [ -n "$ZSH_NAME" ]; then
    THIS_SCRIPT="$0"
else
    echo "The meta-lxatac oe-init-build-env can only be sourced by bash and zsh."
    echo "For other shells (e.g. fish) use a workaround like the following:"
    echo
    echo "  exec bash -c \"source oe-init-build-env && exec fish\""
    exit 1
fi

DIR=$( dirname -- "${THIS_SCRIPT}" )
unset THIS_SCRIPT

```